examples/lusid/portfolios/create-reference-portfolio.sql

```diff
@@ -0,0 +1,23 @@
+-- #################### SUMMARY ############################
+
+-- 1. In this query, we crate a Reference Portfolio in LUSID
+
+-- #########################################################
+
+-- Defining scope and code variables
+@@portfolioScope = select 'IBOR';
+@@portfolioCode1 = select 'uk-equity-model';
+
+-- Define the portfolio data
+@create_portfolio = select 'Reference' as PortfolioType,
+@@portfolioScope as PortfolioScope,
+@@portfolioCode1 as PortfolioCode,
+@@portfolioCode1 as DisplayName,
+'' as Description,
+#2000-01-01# as Created,
+'GBP' as BaseCurrency;

```

Comment:
This is an example of where we could improve style (via Pretty print or manually)
```
@create_portfolio = 
select 
   'Reference' as PortfolioType,
   @@portfolioScope as PortfolioScope,
   @@portfolioCode1 as PortfolioCode,
   @@portfolioCode1 as DisplayName,
   '' as Description,
   #2000-01-01# as Created,
   'GBP' as BaseCurrency;
```